### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mariano54**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,20 +88,18 @@ provided configuration with `pre-commit install`.
...
-2. Run blackd in a terminal
-3. Install BlackConnect plugin
-4. Set to run python black on save
-5. Set line length to 120
-6. Install the linters in the root directory
+1. Run blackd in a terminal
+2. Install BlackConnect plugin
+3. Set to run python black on save
+4. Set line length to 120
+5. Install the linters in the root directory
```
Comment (altendky) [inline CONTRIBUTING.md:97]: I'm not sure what `5.` means and I personally can't comment on the rest as the closest I come to black integrations is a `ctrl+b` shortcut.  Do you have to have the venv activated in the console where you launch pycharm?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -86,20 +88,18 @@ provided configuration with `pre-commit install`.
...
 
 ## Testnets and review environments
 
-With the launch of `1.0.0` we will begin running an official `testnet`.
-Prior to the release of `1.1.0` there will be two running test nets. `testnet` and `transaction-beta-testnet`. The `transaction-beta-testnet` testnet will be a beta of the pending 1.1 release, which will enable transactions on the chia blockchain.
-Following the release of `1.1.0`, the official `testnet` will include all changes that have been accepted to the current release branch.
+The current official testnet is testnet10. Look at `chia/util/initial_config.yaml` to see the configuration parameters
+for each testnet. Information on how to use the testnet can be found in the wiki.
```
Comment (altendky) [inline CONTRIBUTING.md:102]: Maybe a link to the wiki?

**Review by altendky — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mariano54) [reply CONTRIBUTING.md:97]: not sure what 5 means. I just left it there since I didn't know what it meant

File: CONTRIBUTING.md (hunk 2 above)
Comment (mariano54) [reply CONTRIBUTING.md:102]: These links change a lot so I rather avoid adding a link

Comment (altendky) [reply CONTRIBUTING.md:102]: Fair, another check to add.  A link checker.

**Review by altendky — APPROVED** [review]

**Pull request merged**